static/app/views/performance/landing/widgets/widgets/lineChartListWidget.tsx

```diff
@@ -178,25 +209,58 @@ export function LineChartListWidget(props: Props) {
               setSelectedIndex={setSelectListIndex}
               items={provided.widgetData.list.data.map(listItem => () => {
                 const transaction = listItem.transaction as string;
+
+                const additionalQuery: Record<string, string> = {};
+
+                if (props.chartSetting === PerformanceWidgetSetting.SLOW_HTTP_OPS) {
+                  additionalQuery.breakdown = 'http';
+                  additionalQuery.display = 'latency';
+                } else if (props.chartSetting === PerformanceWidgetSetting.SLOW_DB_OPS) {
+                  additionalQuery.breakdown = 'db';
+                  additionalQuery.display = 'latency';
+                } else if (
+                  props.chartSetting === PerformanceWidgetSetting.SLOW_BROWSER_OPS
+                ) {
+                  additionalQuery.breakdown = 'browser';
+                  additionalQuery.display = 'latency';
+                } else if (
+                  props.chartSetting === PerformanceWidgetSetting.SLOW_RESOURCE_OPS
+                ) {
+                  additionalQuery.breakdown = 'resource';
+                  additionalQuery.display = 'latency';
+                }
+
                 const transactionTarget = transactionSummaryRouteWithQuery({
                   orgSlug: props.organization.slug,
                   projectID: listItem['project.id'] as string,
                   transaction,
                   query: props.eventView.getGlobalSelectionQuery(),
+                  additionalQuery,
                 });
+
+                const fieldString = getAggregateAlias(props.fields[0]);
+
+                const valueMap = {
+                  [PerformanceWidgetSetting.MOST_RELATED_ERRORS]: listItem.failure_count,
+                  [PerformanceWidgetSetting.MOST_RELATED_ISSUES]: listItem.issue,
+                  slowest: getPerformanceDuration(listItem[fieldString] as number),
+                };
+                const rightValue =
+                  valueMap[isSlowestType ? 'slowest' : props.chartSetting];
+
                 switch (props.chartSetting) {
                   case PerformanceWidgetSetting.MOST_RELATED_ISSUES:
                     return (
                       <Fragment>
-                        <GrowLink to={transactionTarget}>
+                        <GrowLink to={transactionTarget} className="truncate">
```

Comment:
Again, should we be using `className` still?

Reply:
Yes because the truncation below is separate, it handles truncating past a certain line length, but for a flex container that grows we also need this class to avoid overflow, or at least this is the fix I'm currently using to make sure the mini-lists don't overflow outside of the widget.